ssg/yaml.py

```diff
@@ -48,9 +49,20 @@ def _open_yaml(stream, original_file=None, substitutions_dict={}):
 
         return yaml_contents
     except Exception as e:
+        count = 0
         _file = original_file
         if not _file:
             _file = stream
+        with open(_file, "r") as e_file:
+            lines = e_file.readlines()
+            for line in lines:
+                count = count + 1
+                if re.match(r"^\t+\s+", line) or re.match(r"^\s+\t+", line):
+                    print("Exception while handling file: %s" % _file, file=sys.stderr)
+                    print("TabIndentationError: Line %s contains tabs instead of spaces:" % count)
+                    print("%s\n\n" % repr(line.strip("\n")))
```

Comment:
Please add `, file=sys.stderr` to all `print` statements, otherwise the error message won't be visible for the user.

Reply:
@jan-cerny thanks. Fixed it!